modules/installation-gcp-user-infra-init-bootstrap.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_gcps_user_infra/installing-gcp-user-infra.adoc
+
+[id="installation-gcp-user-init-infra-bootstrap_{context}"]
+= Initializing the bootstrap machine on GCP with user-provisioned infrastructure
```

Comment:
"Initializing the bootstrap machine" isn't quite right (to me it sounds like "Create the bootstrap machine", which is a prereq of this module).  How about "Wait for bootstrap completion on GCP with user-provisioned infrastructure" to echo `wait-for bootstrap-complete`?  Or, to be extra-clear: "Wait for bootstrap completion and remove bootstrap resources on GCP with user-provisioned infrastructure".  I'm fine with either of those.